security/clamav/src/opnsense/scripts/OPNsense/ClamAV/freshclam.sh

```diff
@@ -26,10 +26,11 @@
 # SUCH DAMAGE.
 
 PIDFILE="/var/run/clamav/freshclam-init.pid"
-DBFILE="/var/db/clamav/main.cvd"
+CVDDBFILE="/var/db/clamav/main.cvd"
+CLDDBFILE="/var/db/clamav/main.cld"
 COMMAND="${1}"
 
-if [ -f ${DBFILE} ]; then
+if [ -f ${CVDDBFILE} -o -f ${CLDDBFILE} ]; then
```

Comment:
Is this correct, only one of these files must be there?

Reply:
Yes, only one.

Reply:
cvd is compressed and should go to cld but this does not work like the author state in some mailing lists. 
For me it was main.cvd and daily.cld ... now I removed the folder and rerun, now it's only cvd.
@kekek2 Is you main cld or cvd?

Reply:
I have "main.cvd". @evbevz have "main.cld"

When you first start freshclam download .CVD. When updating freshclam is trying to download the difference. If the difference is downloaded, the base is unpacked, apply the diff, and still is .CLD. If the difference is not found, download again .CVD. If there is an update for the database "main", then freshclam will be able to download the diff to it, it becomes the "main.cld".